packages/mdc-fab/README.md

```diff
@@ -99,6 +101,7 @@ MDC FAB uses [MDC Theme](../mdc-theme)'s `secondary` color by default. Use the f
 Mixin | Description
 --- | ---
 `mdc-fab-accessible($container-color)` | Changes the FAB's container color to the given color, and updates the FAB's ink and ripple color to meet accessibility standards.
+`mdc-fab-extended-fluid()` | Makes the Extended FAB fluid to container, such as screen width or the layout grid.
```

Comment:
Where'd you get the word fluid from? I understand its for responsive layouts, but all the mixin does is make the fab full-width.

Reply:
Yah, that is true. I got this term from Spec

https://material.io/design/components/buttons-floating-action-button.html#extended-fab